internal/types/options.go

```diff
@@ -0,0 +1,22 @@
+package types
```

Comment:
Again, not a fan of 'types' as a package. And options really should remain in capability. It's an implementation detail that should not be exposed.

Reply:
removed 👍